starlingcaptureapi/exif.py

```diff
@@ -72,4 +72,8 @@ def _to_rational_str(self, deg_tuple):
     def _to_rational(self, number):
         """Convert number to rational, (numerator, denominator)."""
         f = Fraction(number)
+        # Rounding ensures that the result fits in 4 bytes (es per EXIF spec),
```

Comment:
```suggestion
        # Rounding ensures that the result fits in 4 bytes (as per EXIF spec),
```

(Not sure if type-o or supposed to be `es per`)